samples/bluetooth/central_esl/Kconfig

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic ESL BLE GATT service client sample"
+
+config BT_ESL_CLIENT_THREAD_STACK_SIZE
+	int "Thread stack size"
+	default 1024
+	help
+	  Stack size used in each of the two threads
+
+config BT_ESL_SECURITY_ENABLED
+	bool "Enable security"
+	select BT_SMP
+	select BT_BONDABLE
+	select BT_BONDING_REQUIRED
+	help
+	  "Enable BLE security for the ESL service"
+
+config BT_ESL_AP_AUTO_MODE
+	bool "AP auto onboarding Tag mode"
+	help
+	  "Enable,AP will auto scan and configure tag. Disable,AP will only accept shell command"
+
+if BT_ESL_AP_AUTO_MODE
+config BT_ESL_AP_AUTO_TAG_PER_GROUP
+	int "Tag per group"
+	default ESL_CLIENT_MAX_RESPONSE_SLOT_BUFFER
+	help
+	  "How many tags should keep in same group for auto onboarding.
+	   Tag number exceeds this number should be in next group.
+	   Otherwise, response key is not enough for all tags in same group.
+	   How to choose this number depends on what scenario we want to show.
+	   Less or equal to ESL_CLIENT_MAX_RESPONSE_SLOT_BUFFER demostrates decrypts response EAD on-the-fly.
+	   Greater than ESL_CLIENT_MAX_RESPONSE_SLOT_BUFFER demostrates ESL AP able to control large amount of tags at the same time."
+
+config BT_ESL_AP_GROUP_PER_BUTTON
+	int "Group per button"
+	default 3
+	help
+	  "How many groups should be controlled by one button.
+	   If this number is greater than 1, AP will control multiple groups by one button.
+	   Otherwise, AP will control one group by one button."
```

Comment:
```suggestion
	   If this number is greater than 1, an AP can control multiple groups by one button.
	   Otherwise, an AP controls one group by one button."
```